tidb-server/main.go

```diff
@@ -433,20 +432,32 @@ func createServer() {
 }
 
 func setupSignalHandler() {
-	sc := make(chan os.Signal, 1)
-	signal.Notify(sc,
+	dumpSignalChan := make(chan os.Signal, 1)
+	signal.Notify(dumpSignalChan, syscall.SIGUSR1)
+	go func() {
+		buf := make([]byte, 1<<16)
+		for {
+			sig := <-dumpSignalChan
+			if sig == syscall.SIGUSR1 {
```

Comment:
We don't need to check the signal.

Reply:
maybe future we need other signal?

Reply:
this if and sig local variable is in popose